angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/radio_control_value_accessor.ts

```diff
@@ -204,6 +217,33 @@ export class RadioControlValueAccessor extends BuiltInControlValueAccessor imple
     };
   }
 
+  /** @nodoc */
+  override setDisabledState(isDisabled: boolean): void {
+    /**
```

Comment:
I didn't try and fully grasp what this is going on about, but is there something we should capture as Angular usage guidance for radio buttons from what is discussed here?

Reply:
After spending a good chunk of time trying to understand the consequences of our radio CVA not respecting `callSetDisabledState = 'whenDisabledForLegacyCode'`, I've come to the conclusion that it's not even worth documenting. What they're doing in the radio CVA doesn't seem to make any sense. Below is a matrix of the behaviors in different cases:
![image](https://github.com/ni/nimble/assets/7282195/993490fb-b087-409a-8d93-4b5ecc81be31)


- The pre-v15 (buggy) behavior was that `setDisabledState` wouldn't be called during initialization if the initial value of `disabled` was `false`. 
- The [fixed logic](https://github.com/angular/angular/blob/b76b3c619bc2ff9ffdd436559a7550398a810b10/packages/forms/src/directives/shared.ts#L78) additionally calls `setDisabledState` when `disabled` is `false`. This depends on a `callSetDisabledState` flag, whose value defaults to `'always'`.
- If a client configures `FormsModule` or `ReactiveFormsModule` with `callSetDisabledState = 'whenDisabledForLegacyMode'`, they get the "legacy" behavior, i.e. `setDisabledState` is called for the control only "when disabled."
- For reasons, the radio CVA could not use the new, fixed behavior. It was important for it to continue _NOT_ calling `setDisabledState` if the initial value of `disabled` was `false`. To accomplish this, they overrode `setDisabledState` to [make it a no-op](https://github.com/angular/angular/blob/b76b3c619bc2ff9ffdd436559a7550398a810b10/packages/forms/src/directives/radio_control_value_accessor.ts#L222) on the initial call, if `disabled` was `false`. In other words, the radio CVA uses legacy mode behavior.
- However, if the legacy mode flag (`'whenDisabledForLegacyMode'`) is set, the radio CVA's `setDisabledState` is _not_ a no-op when `disabled` is `false`. Essentially the legacy mode flag causes it to use non-legacy behavior. This makes zero sense, and there is no stated use case for wanting the radio CVA to ever use non-legacy behavior. I can only assume this is a bug (with extremely limited impact).

Reply:
So configuring `[SetDisabledStateOption](https://angular.io/api/forms/SetDisabledStateOption)` is something users can do, like this example: https://stackoverflow.com/a/75209669

Sounds like we ignore that configuration. Is that not worth capturing in the Usage Docs with a brief explanation (i.e. Nimble Radio Button just supports mode `always`)?

Reply:
Angular's `NgForm` class handles the `whenDisabledForLegacyCode` option for all CVAs other than Radio CVA. Based on the comment in Angular's Radio CVA, the Radio behaves differently than all other CVAs because the form models the entire group of radio buttons as a single `FormControl`, which means there isn't a way to control the disabled state for individual radios.

Therefore, to answer your questions:
- There already is a test in our forked `value_accessor_integration.spec.ts` for `callSetDisabledState: 'whenDisabledForLegacyCode'`.
- I expect that mode to work just fine for our other CVAs

Reply:
I've converted the radio group story to an MDX file and added a statement about the CVA behavior.